### ISSUE TITLE
Zoom object

### ISSUE THREAD

**Issue by tekboxs** [MasahideMori1111/simple_3d#1]
For me it was great, the only thing i missed was the possibility to zoom in on the object using gestures, is there any support for that?

**Comment by MasahideMori-SimpleAppli** [comment]
Hi @tekboxs!

Thank you for your feedback.

The current renderer does not support zoom operations,
but I will consider whether it can support zoom.
I didn't notice it because I usually only use this package on PC.

Thank you!

**Comment by MasahideMori-SimpleAppli** [comment]
Hi @tekboxs !

A little time ago I uploaded a new version of the renderer.
A new renderer for simple_3d, version 6.0.0 (https://pub.dev/packages/simple_3d_renderer) 
supports pinch gestures and mouse wheel zoom.
Please, try it.
Thank you very much!

**Comment by tekboxs** [comment]
Thanks for your attention, but unfortunately on mobile, it was complicated to use the gestures, I feel like the zoom in was much faster than the zoom out. This makes it difficult to control the object naturally.

**Comment by MasahideMori-SimpleAppli** [comment]
Hi @tekboxs , Thank you for your feedback.
I would like to add a method that allows for more detailed control of the camera.
The current linear control seems to have poor usability.
I'm planning on adding a dedicated argument.

**Comment by MasahideMori-SimpleAppli** [comment]
Hi @tekboxs ,
I just released version 7.0.0 of simple_3d_renderer.
The new version allows the user to freely define the zoom control of the camera.
If you want to change the pinch-in and pinch-out movement, for example, create a subclass like the one below and give it as an argument to Sp3dRenderer.
```dart
import 'package:flutter/material.dart';
import 'package:simple_3d_renderer/simple_3d_renderer.dart';

@immutable
class CustomZoomCtrl extends Sp3dCameraZoomController{

  @override
  void apply(Sp3dCamera camera, double zoomV, bool isMouse) {
    // pinch-in or pinch-out
    if(zoomV > 0) {
      // Make it behave differently than the default.
      camera.focusLength = camera.focusLength + zoomV - 50;
    }
    else{
      camera.focusLength = camera.focusLength + zoomV;
    }
    // Negative values are not allowed as they invert the calculation.
    if (camera.focusLength < 0) {
      camera.focusLength = 1;
    }
  }

}
```
The best calculation method probably depends on what you want to make.
I hope you find the best way! Thanks.
